saw-python/src/rcs.c

```diff
@@ -0,0 +1,13 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdint.h>
+
+uint32_t RCS(uint32_t bits, uint32_t shift) {
+  shift %= 64;
```

Comment:
That `64` should be `32`. Or, for consistency w/ the line two below, we could write `sizeof(bits)*8`.